lib/puppet-lint/plugins/check_parameter_documentation.rb

```diff
@@ -12,11 +13,15 @@ def check
              dtok.value =~ /\A\s*@param (?:\[.+\] )?([a-zA-Z0-9_]+)(?: +|$)/
             doc_params << $1
           end
+
+          is_private = true if dtok.value =~ /\A\s*@api private\s*$/
```

Comment:
That regex won't catch something like an extra space. Should it be `/\A\s*@api +private\s*$/`?

Reply:
Good point. It might be good to really use puppet-strings at some point instead of regexes, but this is the quickest way.